src/features/app/services/localeMiddleware.js

```diff
@@ -0,0 +1,13 @@
+export const applyLocaleInterceptor = (client, isAuthenticated, locale) => {
```

Comment:
Why does this function receive the `client` as a parameter instead of importing it directly on the file?

Reply:
good catch, we can do that for sure 👌